DESCRIPTION

```diff
@@ -67,10 +67,11 @@ VignetteBuilder:
     knitr
 Config/testthat/edition: 3
 Config/Needs/website:
-    usethis
+    usethis,
+    servr
```

Comment:
for auto-linking if servr once gains a pkgdown website.

Reply:
We could also add servr to suggests.

Reply:
Why would we add it to suggests?

Reply:
I see we're just referring to it as a helper, not recommending it. So no need to add to suggests.